sources/PanelButton.cpp

```diff
@@ -68,6 +73,14 @@ PanelButton::CreatePanel(int8 which, const BView* target, uint32 flavors,
 		panel = new BFilePanel(B_OPEN_PANEL, &msgr, NULL, flavors, false, NULL,
 			filter);
 
-		panel->Window()->SetTitle(pairs[which].locale);
+		BString title = B_TRANSLATE("Filer");
+		title += ": ";
+
+		if (prefix != NULL)
+			title << prefix << ' ';
```

Comment:
Now I think I understand your concern. The parameter `pairs` will be substituted with the `sFileTypes` or `sActions` argument from `RuleRunner`, and the C strings in these two arrays are already localized and used for the popup menus in TestView and ActionView, respectively. The file panel titles will simply reuse the translated strings.